microsoft/testsuites/acc/bvt.py

```diff
@@ -0,0 +1,191 @@
+            ),
+            keys_location=["https://apt.llvm.org/llvm-snapshot.gpg.key"],
+        )
+
+        # 3. Get Ubuntu 20.04 packages from Microsoft prod repo
+        #
+        # < echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/18.04/\
+        #   prod bionic main" |
+        #   sudo tee /etc/apt/sources.list.d/msprod.list >
+        # < wget -qO - https://packages.microsoft.com/keys/microsoft.asc
+        #   | sudo apt-key add - >
+
+        node.os.add_repository(
+            repo=(
+                "deb [arch=amd64] https://packages.microsoft.com/"
+                f"ubuntu/{os_version}/prod {node.os.information.codename} main"
+            ),
+            keys_location=[
+                "https://packages.microsoft.com/keys/microsoft.asc",
+            ],
+        )
+
+        # Verify if Intel DCAP client installed.
+        #
+        # <get dmesg dmesg | grep -i sgx>
+
+        dmesg = node.tools[Dmesg]
+        if "sgx" not in dmesg.get_output():
+            node.os.install_packages("dkms")
+            wget = node.tools[Wget]
+            driver_path = wget.get(
+                url="https://download.01.org/intel-sgx/sgx-dcap/1.7/"
+                f"linux/distro/ubuntu{os_version}-server/sgx_linux_x64_driver_1.35.bin",
+                filename="sgx_linux_x64_driver.bin",
+                executable=True,
+            )
+            node.execute(driver_path, sudo=True)
+
+        # Install other packages
+        # < sudo apt -y install clang-10 libssl-dev gdb libsgx-enclave-common
+        #   libsgx-quote-ex libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev \
+        #   az-dcap-client open-enclave >
+
+        if os_version == "18.04":
+            libprotobuf = "libprotobuf10"
+        else:
+            libprotobuf = "libprotobuf17"
+
+        node.os.install_packages(
+            f"clang-10 libssl-dev gdb libsgx-enclave-common libsgx-quote-ex "
+            f"{libprotobuf} libsgx-dcap-ql libsgx-dcap-ql-dev "
+            "az-dcap-client open-enclave"
+        )
+        # REMOVE install_cmd = "apt -y install clang-10 libssl-dev gdb
+        # libsgx-enclave-common libsgx-quote-ex libprotobuf10
+        # libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client
+        #  open-enclave "
+        # stdout = node.execute(install_cmd, sudo=True)
+
+        # Copy Samples from the Binary
+        #
+        # <cp -r /opt/openenclave/share/openenclave/samples ~/mysamples>
+
+        samples_folder = node.get_working_path() / "mysamples"
+        copy_cmd = "cp -r /opt/openenclave/share/openenclave/samples " + str(
+            samples_folder
+        )
+        node.execute(copy_cmd, shell=True)
+
+        # Run Hello World and Remote Attestation
+        helloworld_dir = samples_folder / "helloworld"
+        attestation_dir = samples_folder / "attestation"
+        source_command = ". /opt/openenclave/share/openenclave/openenclaverc"
+        fail_msg = "HELLO WORLD TEST FAILED"
+
+        node.tools[Make]
```

Comment:
Why is this required? If `make` is a prerequisite, you can directly call install. @squirrelsc Is there a way to add a tool as a dependency?